nixos/modules/services/hardware/undervolt.nix

```diff
@@ -21,6 +26,9 @@ let
 
     temp-bat = cfg.tempBat;
     temp-ac = cfg.tempAc;
+
+    power-limit-long = mkPLimit cfg.p1.limit cfg.p1.window;
+    power-limit-short = mkPLimit cfg.p2.limit cfg.p2.window;
```

Comment:
why expose these as `p1` and `p2` rather than just reusing the argument name from the underlying tool?

Reply:
The names of the short option are p1 & p2 which are also the terms used in hardware forums etc. I don't think I've ever heard 'long' & 'short' for these.

Reply:
I agree that we should use the original name of the option. Its not our place to rename it, it only adds to the confusion. If you mention the term "P1" and "P2" in the relevant descriptions, searchability should be just as good.

If feel strongly that the names should be kept p1 and p2, the description should at least mention the name of the underlying configuration options.

Reply:
These are the original names of the options: https://github.com/georgewhewell/undervolt/blob/13fa33d05ae1d1306a2d3e2cd921028c1eb785d6/undervolt.py#L376